### DISCUSSION TITLE
Account flagged, support not responding

### DISCUSSION THREAD

**Discussion by just4educt** [community/community#138008] [GitHub Education]
### Hi everyone,

Hi, my account got suspended. I opened a support ticket the same day but got no answer after over 1 week, I cannot work anymore

Is there any option to get an answer to my support ticket

Ticket: #2977454

**Comment by Shriharsh-Deshmukh** [comment]
You could try reaching out to GitHub Support via their social media channels, like Twitter, where they sometimes respond faster. Make sure to reference your ticket number #2977454 when contacting them.

I hope you get your issue resolved soon!

**Reply by just4educt** [reply]
Thanks, but I didn't find the twitter account

**Reply by centxv** [reply]
> Thanks, but I didn't find the twitter account

could you please share there twitter account 
im also facing same issue like you had

**Reply by Shriharsh-Deshmukh** [reply]
**Github Twitter/ X Account:** https://x.com/github?

**Reply by centxv** [reply]
> **Github Twitter/ X Account:** https://x.com/github?

Thanks! Could you let me know if you got any replies via Twitter?
I heard resubmitting support with a different working GitHub account might
help reference the original ticket. I did that too.

**Comment by Zouhair180** [comment]
[GitHub Community] Birtday (Discussion #138090) ([مجتمع GitHub] عيد الميلاد (مناقشة رقم 138090))

**Comment by ch10874** [comment]
Hi, @just4educt 
I can't find your ticket-#2977454.

[Support Ticket](https://support.github.com/request)

**Reply by just4educt** [reply]
> Hi, @just4educt I can't find your ticket-#2977454.
> 
> [Support Ticket](https://support.github.com/request)

It's from my suspended account @sakanjo

**Comment by just4educt** [comment]
...

**Comment by KnightGOKU** [comment]
I’m still encountering the same issue. My ticket has been pending for over a week with no response yet. :( Could anyone advise on how I might expedite the resolution?

**Comment by recluzedev1** [comment]
My account has been flagged since july 24 No response from github support I have generated several tickets

**Comment by baharbaba** [comment]
I've been waiting for a response for 1 month. A week ago, I received a message saying "We have not forgotten about you..." but now I feel like I have been forgotten. From now on, I'll be tagging @github-staff in every discussion that has this issue. I guess you need to be Karen to get a problem solved.

**Reply by centxv** [reply]
> I've been waiting for a response for 1 month. A week ago, I received a message saying "We have not forgotten about you..." but now I feel like I have been forgotten. From now on, I'll be tagging @github-staff in every discussion that has this issue. I guess you need to be Karen to get a problem solved.

where did you use tagging, mine is getting closer  to 2 months

**Reply by baharbaba** [reply]
i did tagged github staff in every discussion that has the waiting issue you can search on google the github support just doesnt wants to help us so im thinking it would be helpful to spam them

**Comment by the-aliXon** [comment]
Dear @github-staff & Everyone , 
 
I've found this topic, and I have a similar problem for this account, @the-aliXon, which is totally 'clean', as I just created it last week and hadn't even been doing anything on GitHub besides fine-tuning the settings for it by the time it got flagged as 'Spammy' for an unjustified reason.
 
I opened a Support Ticket (#3020732) providing all the necessary details, but GitHub Support is non-responsive up to now.
 
I started a Community discussion (#140296) titled "Seeking Help: Newly-Created GitHub Account Flagged as 'Spammy' Without Cause, Support Non-Responsive", where I also detailed the situation and mentioned that Ticket, but it turns out my discussion cannot even be seen by anyone besides myself.
 
I am not even certain this post of mine will be visible to others...
 
This is extremely frustrating.

**Reply by the-aliXon** [reply]
I have just opened a new ticket regarding this--#4181958. As it turns out, because of this unjustified flagging, I cannot even update my billing information, nor can I upgrade to GitHub Pro.

**Comment by omalizadeh** [comment]
It's been a month now since my account got flagged as spammy without giving any specific reason. no response on ticket either. ticket number: #3041966
@github-staff

**Reply by omalizadeh** [reply]
@queenofcorgis Could you please help?

**Comment by omar-m-z-dotcom** [comment]
### unable to use copilot on vscode and delete my organization because my account has been flagged by mistake

Hello, I'am currently facing some issues with deleting the organization "Omar-Business-intelligence-repos" which I planned to use for grouping my personal github repos (not for an actual organization).
Little did I know that this was a very bad idea and there were smarter ways to do grouping, hence I tried to delete this organization but wasn't able to do it. 
Because my account has been flagged by mistake I tried to contact customer support about this but I couldn't because my subscription on github is free and I believe this issue is interfering with my use of github copilot on vscode.
I would appreciate it if I can get a response about this issue as soon as possible.

also here is some footage and some logs that could help in solving the problem:

![WhatsApp Image 2024-11-15 at 02 56 04_a32dedd4](https://github.com/user-attachments/assets/05b16ae9-8d98-445f-9c68-39abb4b4b663)
![WhatsApp Image 2024-11-15 at 02 31 59_8c8e603e](https://github.com/user-attachments/assets/ae8f13bc-0812-4085-9ec9-4cfa028ac5e8)
![WhatsApp Image 2024-11-15 at 02 29 54_106f5aa8](https://github.com/user-attachments/assets/1e10e1ab-6163-4f00-8a5c-ac1da114b468)

1. output on vscode from Developer: GitHub Copilot chat Diagnostics:
  ## GitHub Copilot Chat

- Extension Version: 0.23.2024102903 (prod)
- VS Code: vscode/1.95.3
- OS: Windows

## Network

User Settings:
json
  "github.copilot.advanced": {
    "debug.useElectronFetcher": true,
    "debug.useNodeFetcher": false
  }


Connecting to https://api.github.com/:
- DNS ipv4 Lookup: 140.82.121.6 (6 ms)
- DNS ipv6 Lookup: Error: getaddrinfo ENOTFOUND [api.github.com](http://api.github.com/)
- Electron Fetcher (configured): HTTP 200 (213 ms)
- Node Fetcher: HTTP 200 (203 ms)
- Helix Fetcher: HTTP 200 (214 ms)

Connecting to https://api.githubcopilot.com/_ping:
- DNS ipv4 Lookup: 140.82.113.22 (8 ms)
- DNS ipv6 Lookup: Error: getaddrinfo ENOTFOUND [api.githubcopilot.com](http://api.githubcopilot.com/)
- Electron Fetcher (configured): HTTP 200 (897 ms)
- Node Fetcher: HTTP 200 (1425 ms)
- Helix Fetcher: HTTP 200 (421 ms)

## Documentation

In corporate networks: [Troubleshooting firewall settings for GitHub Copilot](https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-firewall-settings-for-github-copilot).

2. output on vscode from github copilot logs:
   2024-11-15 02:53:31.756 [info] Logged in as omar-m-z-dotcom
2024-11-15 02:53:32.066 [warning] Invalid copilot token: missing token: 403
2024-11-15 02:53:32.066 [info] copilot token chat_enabled: undefined, sku:
2024-11-15 02:53:32.076 [info] Logged in as omar-m-z-dotcom
2024-11-15 02:53:32.360 [warning] Invalid copilot token: missing token: 403
2024-11-15 02:53:32.360 [error] GitHub Copilot could not connect to server. Extension activation failed: "Contact Support. You are currently logged in as omar-m-z-dotcom."
2024-11-15 02:53:32.360 [warning] [LanguageModelAccess] LanguageModel/Embeddings are not available without auth token
2024-11-15 02:53:32.360 [error] Error: Contact Support. You are currently logged in as omar-m-z-dotcom.
    at tC._authShowWarnings (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:412:3282)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at tC.getCopilotToken (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:412:2135)
    at UR.getCopilotToken (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:282:39062)
    at gC._getAuthSession (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:447:2552)
    at t (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:447:2015)
2024-11-15 02:53:32.361 [warning] [LanguageModelAccess] LanguageModel/Embeddings are not available without auth token
2024-11-15 02:53:32.361 [error] Error: Contact Support. You are currently logged in as omar-m-z-dotcom.
    at tC._authShowWarnings (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:412:3282)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at tC.getCopilotToken (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:412:2135)
    at UR.getCopilotToken (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:282:39062)
    at gC._getAuthSession (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:447:2552)
    at t (c:\Users\triro\.vscode\extensions\github.copilot-chat-0.23.2024102903\dist\extension.js:447:631)
3. name: omar mohamed
4. username: omar-m-z-dotcom
5. email: triro765@gmail.com
6. organization to be deleted: Omar-Business-intelligence-repos

Thank you.
